src/main/java/frc/robot/controls/Driver.java

```diff
@@ -41,4 +46,48 @@ public Driver(Robot robot)
 	// METHODS //
 
 	// Add methods here which return values for various robot controls by reading the controllers.
+
+    /* =====================================
+    getThrottle() will return the current
+    value of the Y-Axis of the left stick on the controller
+    It will also handle adjusting
+    the reading according to the deadzone,
+    and according to the max throttle speed.
+    ===================================== */
+    public double getThrottle()
+    {
+        double throttleValue = 0;
+        
+        throttleValue = -dualshock.getY();
+        throttleValue *= Constants.MAX_THROTTLE_SPEED;
```

Comment:
If you want, you could do these all in one line:
```
double throttleValue = -dualshock.getY() * Constants.MAX_THROTTLE_SPEED;
```
I think it might run a little faster unless the Java compiler is super smart :P

Reply:
I'm not able to do this all on one line since the deadzone check has been moved to between reading the value from the joystick and scaling the input.